src/Resources/config/compatibility/services_6.3.xml

```diff
@@ -9,7 +9,7 @@
         <service id="Kiener\MolliePayments\Compatibility\Storefront\Route\PaymentMethodRoute\Voucher\HideVoucherPaymentMethodRoute63"
                  decorates="Shopware\Core\Checkout\Payment\SalesChannel\PaymentMethodRoute">
             <argument type="service" id="Kiener\MolliePayments\Compatibility\Storefront\Route\PaymentMethodRoute\Voucher\HideVoucherPaymentMethodRoute63.inner"/>
-            <argument type="service" id="service_container"/>
+            <argument type="service" id="Shopware\Core\Checkout\Cart\SalesChannel\CartService"/>
```

Comment:
If you decorates core class and you don't want to add parent class argument in your service then you can use parent keyword with decorates EX.

`<service id="Kiener\MolliePayments\Compatibility\Storefront\Route\PaymentMethodRoute\Voucher\HideVoucherPaymentMethodRoute63"
        decorates="Shopware\Core\Checkout\Payment\SalesChannel\PaymentMethodRoute">`


`<service id="Kiener\MolliePayments\Compatibility\Storefront\Route\PaymentMethodRoute\Voucher\HideVoucherPaymentMethodRoute63"
        decorates="Shopware\Core\Checkout\Payment\SalesChannel\PaymentMethodRoute"
        parent="Shopware\Core\Checkout\Payment\SalesChannel\PaymentMethodRoute">`

And use class construct like this 

public function __construct
   (
       AbstractFormRoute $formRoute,
       EntityRepositoryInterface $eecomSwagCmsExtensionsRepository,
       EntityRepositoryInterface $eecomCustomFormSubmitRepository,
       EntityRepositoryInterface $formRepository